### ISSUE TITLE
Upload your portfolio

### ISSUE THREAD

**Issue by sudip-mondal-2002** [sudip-mondal-2002/HactoberFest21#10]
You can add your portfolio by creating a folder with a unique name and keep your codes inside that
* This can be using HTML, CSS
* This can be an app
* anything else(like markdown)

**Comment by tj8016** [comment]
Hello ,  i loved to contribute in this can i upload website

**Comment by sudip-mondal-2002** [comment]
Sure, welcome! Fork this repository, create a folder, upload your code and create a pull request

**Comment by ruturaj-gh** [comment]
hello im beginner html and css learner created a page 
can i contribute my local page

**Comment by sudip-mondal-2002** [comment]
Yes sure, this repo is made for beginners to come and contribute

**Comment by Shreyajaiswal11** [comment]
I would also like to contribute

**Comment by sudip-mondal-2002** [comment]
Yeah go on!!

**Comment by passionlord** [comment]
can I contribute by adding my basic html css portfolio page ?

**Comment by sudip-mondal-2002** [comment]
Yeah Sure!!

**Comment by Hacker-sagar1** [comment]
You can add your portfolio by creating a folder with a unique name and keep your codes inside that

This can be using HTML, CSS
This can be an app
anything else(like markdown)

**Comment by sudip-mondal-2002** [comment]
Sure just create a pull request

**Comment by nishantkumar1308** [comment]
I want to create my portfolio .please assign it to me.

**Comment by sudip-mondal-2002** [comment]
Sure!! go on

**Comment by MariamChowdhury** [comment]
Hi, I want to work on this. Can you please assign me this issue?

**Comment by Tanushreesuthar** [comment]
please assign me.

**Comment by sudip-mondal-2002** [comment]
@MariamChowdhury @Tanushreesuthar it's not needed to be assigned but okay

**Comment by nishantkumar1308** [comment]
@sudip-mondal-2002 Kindly merge my pr #293 and give Hacktoberfest-accepted tag on my pr

**Comment by Yash-RajSingh** [comment]
I would really like to contribute to this but is there a time limit for this issue? If not please assign it to me

**Comment by sudip-mondal-2002** [comment]
No there is no time limits

**Comment by Karthick986** [comment]
I made a portfolio in Flutter web, plz assign to me to contribute!

**Comment by aayushsrivastava9760** [comment]
I also want to contribute to it . Please assign me too.

**Comment by Code414** [comment]
I want to contribute. Please assign this to me.
